models/games.models.js

```diff
@@ -60,17 +60,42 @@ exports.addComment = (review_id, body_req) => {
     });
   }
 
-    return db.query(
-        `
+  return db
+    .query(
+      `
         INSERT INTO comments 
         (body, author, review_id, votes , created_at)
         VALUES 
         ($1,$2,$3,$4,$5)
         RETURNING *;
     `,
-        [body, author, review_id, 0, created_at]
-      )
+      [body, author, review_id, 0, created_at]
+    )
     .then((comment) => {
       return comment.rows[0];
     });
 };
+
+exports.updateReview = (review_id, body) => {
+  const { inc_votes } = body;
+
+  if (!inc_votes) {
+    return Promise.reject({
+      status: 400,
+      msg: "Input data format was not correct",
+    });
+  }
+  return this.fetchReviewsByID(review_id).then(()=>{
+    return db.query(
```

Comment:
checking review in this way is unnecessary here. I would suggest taking this out and seeing what you get back from this query if given an invalid review_id, and then checking what you get back from this and how that can be used to Promise.reject with a status code and error message.

Reply:
I have tried to remove the calling of the other function, but it does not give me any error from SQL, it gives me a 201. I did get an error from SQL in the case of the post. but i didnt for this one.